lca_website/ngo/forms.py

```diff
@@ -0,0 +1,27 @@
+from django import forms
+from django.core.validators import validate_email, URLValidator
+from .models import NGO_CATEGORY
+
+
+class SignupForm(forms.ModelForm):
+    name = forms.CharField(label="Name of the NGO", max_length=100)
+    registration_number = forms.CharField(
+        label="Registration number of the NGO", max_length=25
+    )
+    category = forms.ChoiceField(
+        label="Category to which the NGO belongs", choices=NGO_CATEGORY
+    )
+    address = forms.CharField(label="Address of the NGO", max_length=255)
+    contact_number = forms.CharField(
+        label="Contact Number of the NGO", max_length=10, min_length=10
+    )
+    email_address = forms.CharField(
+        label="Email address of the NGO", max_length=255, validators=[validate_email]
+    )
+    website = forms.CharField(
+        label="Website of the NGO",
+        max_length=75,
+        validators=[URLValidator()],
+        required=False,
+    )
+    agree_to_tnc = forms.BooleanField(required=True)
```

Comment:
Can an NGO register without agreeing to TNC?

We might be better off making it implicit, using language like `By registering, you agree to our Terms And Conditions detailed here`.

Reply:
Yes. I agree. 
I used a separate field since it was explicitly mentioned in the requirements.